src/renderer/components/hotbar/hotbar-icon.tsx

```diff
@@ -55,6 +56,21 @@ export class HotbarIcon extends React.Component<Props> {
     }
   }
 
+  get badgeIcon() {
+    const className = "badge";
+    const category = catalogCategoryRegistry.getCategoryForEntity(this.props.entity);
+
+    if (!category) {
+      return <Icon material="bug_report" className={className} />;
+    }
+
+    if (category.metadata.icon.includes("<svg")) {
+      return <Icon svg={category.metadata.icon} className={className} />;
+    } else {
+      return <Icon material={category.metadata.icon} className={className} />;
+    }
+  }
```

Comment:
**nit**: I'd better to use `const iconProps: IconProps = {};` with `if-else` updates and `return <Icon {...iconProps}/>`.